botocore/docs/waiter.py

```diff
@@ -78,6 +80,27 @@ def document_wait_method(section, waiter_name, event_emitter,
     :param include_signature: Whether or not to include the signature.
         It is useful for generating docstrings.
     """
+    waiter_config_members = OrderedDict()
+
+    waiter_config_members['Delay'] = DocumentedShape(
+        name='Delay', type_name='integer',
+        documentation=(
+            '<p>The amount of time in seconds to wait between attempts.</p>'))
```

Comment:
I know this is in the top level description, but it might be good to say what the defaults are for `Delay` and `MaxAttempts` for these operations as well.